src/activities/practice/Practice.qml

```diff
@@ -0,0 +1,692 @@
+                    MouseArea {
+                        anchors.fill: parent
+                        hoverEnabled: true
+                        onHoveredChanged: {
+                             if(containsMouse)
+                                {
+                                 evaluate.scale=1.15
+                             }
+                             else{
+                                 evaluate.scale=1
+                             }
+
+                        }
+                        onClicked: {
+                            evaluate.clicked()
+                            if(firstop.text!="" && operator.text!="" && secondop.text!="" && result.text!="")
+                            {
+                            var input={"firstop":Number(firstop.text),"secondop":Number(secondop.text),"operator":operator.text,"expected_result":Number(result.text),"saved_calculation":false}
+                            firstop.undo()
+                                operator.undo()
+                            secondop.undo()
+                                result.undo()
+                        if(Activity.calculate_result(input))
+                        {
+                            input["correct"]=Activity.calculate_result(input)
+                            audioEffects.play("qrc:/gcompris/src/core/resource/sounds/win.wav")
+                            theModel.insert(0,input);
+                            correctquestions.insert(0,input)
+                        }
+                        else
+                        {
+                            input["correct"]=Activity.calculate_result(input)
+                            audioEffects.play("qrc:/gcompris/src/core/resource/sounds/crash.wav")
+                            theModel.insert(0,input);
+                            wrongquestions.insert(0,input)
+}
+                            }
+                            else
+                                audioEffects.play("qrc:/gcompris/src/core/resource/sounds/crash.wav")
+
+                        }
+
+
+                    }
+
+                }
+                Rectangle {
+                    id:correctanswers
+                    anchors.top: evaluate.bottom
+                    anchors.right: parent.right
+                    anchors.topMargin: 50
+                    anchors.rightMargin: 50
+                    width:350
+                    height: 50
+                    signal clicked
+                    onClicked: NumberAnimation { target: correctanswers; property: "opacity"; from: 0; to: 1; duration: 200 }
+
+                    color: "#53d769"
+                    border.color: Qt.lighter(color, 1.1)
+
+                    GCText {
+                        anchors.centerIn: parent
+                        text: "Correct Answers"
+                    }
+                    MouseArea {
+                        anchors.fill: parent
+                        hoverEnabled: true
+                        onHoveredChanged: {
```

Comment:
I would do the opposite:
in correctanswers, I would do, scale: mouseArea.containsMouse ? 1.15 : 1